manifests/kerberos.pp

```diff
@@ -23,13 +23,24 @@
 # @param root_k5login_principals
 #   Optional Array k5login principals with root privileges
 #
+# @param vaultkeytabkey
+#   # The key to the base64 encoded hostkeytab in vault
+#
+# @param vaultkeytabkey
+#   # The key to the base64 encoded hostkeytab in vault
```

Comment:
It looks like this docstring is listed twice...

```suggestion
# @param vaultkeytabkey
#   # The key to the base64 encoded hostkeytab in vault
```

Reply:
This has been fixed